catalog/camel-catalog/src/main/resources/org/apache/camel/catalog/capabilities.properties

```diff
@@ -0,0 +1,17 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+platform-http=other/platform-http-main
```

Comment:
capabilities are defined as simple properties with the following syntax:

```
${capability-name} = ${kind}/${name}
```

It is enough for the runtimes to create such file and make the location discoverable by overriding the `RuntimeProvider::getCapabilitiesCatalog`

Reply:
Hmm should this file really be in camel-catalog or did you use it for testing purpose

Reply:
What about this file ?

Reply:
it is meant to provide the default capabilities for vanilla camel. 

I only listed `platform-http` as I do not know the others at this stage but I will add things like metrics,,tracing,  health, etc once I get to know what they are. 

Do you think it should be moved somewhere else ?

Reply:
Ah okay so its a bit upside down thinking. The catalog is usually a collection of each individual component information. So `platform-http=other/platform-http-main` would then be in `camel-platform-http-main`, etc. But I can see from your pov, that you want camel-catalog to be vanilla Camel and that it has a set of capabilities. And then in SB catalog, you adjust this for SB runtime. And ditto for Q.

Reply:
yeah, it is a little bit different from how things are done since it is the catalog that "dictates" what is the artifact that provides such feature. Eventually we may find a way to annotate a component to the provider for a capability and then the catalog plugin can auto generate this file but if was a little bit too intrusive so I opted for this simpler way.

Maybe there should be a camel main catalog and this file should be in the related catalog module.